engines/instance_verification/spec/requests/api/connect/v3/subscriptions/systems_controller_spec.rb

```diff
@@ -4,11 +4,94 @@
   describe '#announce_system' do
     let(:instance_data) { '<instance_data/>' }
 
-    it 'saves instance data' do
-      post '/connect/subscriptions/systems', params: { hostname: 'test', instance_data: instance_data }
-      data = JSON.parse(response.body)
-      system = System.find_by(login: data['login'])
-      expect(system.hw_info.instance_data).to eq(instance_data)
+    context 'using RMT generated credentials' do
+      it 'saves instance data' do
+        post '/connect/subscriptions/systems', params: { hostname: 'test', instance_data: instance_data }
+        data = JSON.parse(response.body)
+        system = System.find_by(login: data['login'])
+        expect(system.hw_info.instance_data).to eq(instance_data)
+      end
+    end
+
+    context 'registering the system with SCC' do
```

Comment:
```suggestion
    context 'using SCC generated credentials (BYOS mode)' do
```

Reply:
I think this test needs to move to the scc_proxy spec. 
In the instance_verification spec, we would only need to make sure that a PAYG system that provides a regcode doesn't fail.